packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -189,7 +166,7 @@ class ConfigCommand extends FlutterCommand {
     }
 
     if (argResults == null || argResults!.arguments.isEmpty) {
-      globals.printStatus(usage);
+      globals.printStatus(settingsUsage);
```

Comment:
What I was thinking was actually just adding `settingsUsage` to the `usageFooter`, and keeping the regular `usage` text.

Reply:
That's the previous. Now it acts like `flutter devices`.